PowerControl/Options/FPSLimit.cs

```diff
@@ -13,14 +13,16 @@ public static class FPSLimit
             ResetValue = () => { return "Off"; },
             OptionsValues = delegate ()
             {
-                var refreshRate = DisplayResolutionController.GetRefreshRate();
-                return new string[]
-                {
-                    (refreshRate / 4).ToString(),
-                    (refreshRate / 2).ToString(),
-                    refreshRate.ToString(),
-                    "Off"
-                };
+                int refreshRate = DisplayResolutionController.GetRefreshRate();
```

Comment:
The reason for using `/1`, `/2` and `/4` for FPS limit is that SD does not have VRR display. FPS limit has to be full fractional of the display refresh rate to avoid second degree tearing. The better way is to allow refresh rate to be fluid to change, kind of injecting proper EDID for a given Hz.